nfsstat/README.md

```diff
@@ -16,6 +16,29 @@ The NFSstat check is included in the [Datadog Agent][2] package, so you don't ne
 
 Edit the `nfsstat.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][3], to point to your nfsiostat binary script, or use the one included with the binary installer. See the [sample nfsstat.d/conf.yaml][4] for all available configuration options.
 
+### Log collection
+
+1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Add this configuration block to your `nfsstat.d/conf.yaml` file to start collecting your NFSstat Logs:
+
+   ```yaml
+   logs:
+     - type: file
+       path: /path/to/my/directory/file.log
```

Comment:
You should probably put the default file location here

Reply:
Updated.